src/Utilities/hooks/useEdgeGroups.js

```diff
@@ -0,0 +1,24 @@
+import { useEffect, useState } from 'react';
+import useFeatureFlag from '../useFeatureFlag';
+import { fetchEdgeEnforceGroups } from '../../api';
+
+const useEdgeGroups = (value) => {
+  const [data, setData] = useState(value);
```

Comment:
This is more of a question, but why pass in a false boolean when data can be set in here? I only see it used in one place, and we're returning the value we put inside of it if the featureflag is false.  For that I think itd be better just to set the useState variable here to false, and only change it if its true, then return data.